apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1014,17 +1017,73 @@ static void fs_vfs_sendfile_tc(void)
 	TC_ASSERT_EQ_CLEANUP("stat", ret, OK, close(fd1));
 
 	size = st.st_size;
+    /* case-1: offset = 0 */
 	fd2 = open(dest_file, O_WRONLY | O_CREAT);
 	TC_ASSERT_GEQ_CLEANUP("open", fd2, 0, close(fd1));
 
 	ret = sendfile(fd2, fd1, 0, size);
-	close(fd1);
```

Comment:
why did you remove close(fd1)?
I think it is needed

Reply:
Its not removed, its there down at line 1086 , please check it.

Reply:
Before line 1086, there are many TC_ASSERT_XX macro. If some failure is there, this tc will be returned without "close(fd1)". This is a leak.

Reply:
yes, I missed to check, I will update.